packages/service/src/db.test.ts

```diff
@@ -24,3 +72,85 @@ test("crdts can be persisted", () => {
     ydoc2.getText("codemirror").toJSON()
   );
 });
+
+test("persist equivalent final crdt via different update paths", async () => {
```

Comment:
Different test, lets put it in its own commit so that it doesn't clutter the diff as much